cmd/kubectl-testkube/commands/cloud/connect.go

```diff
@@ -0,0 +1,254 @@
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pterm/pterm"
+	"github.com/skratchdot/open-golang/open"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/config"
+	cloudclient "github.com/kubeshop/testkube/pkg/cloud/client"
+	"github.com/kubeshop/testkube/pkg/cloudlogin"
+	"github.com/kubeshop/testkube/pkg/ui"
+)
+
+const (
+	listenAddr      = "127.0.0.1:8090"
+	docsUrl         = "https://docs.testkube.io/testkube-cloud/intro"
+	tokenQueryParam = "token"
+)
+
+func NewConnectCmd() *cobra.Command {
+	var opts = common.HelmOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "connect",
+		Aliases: []string{"c"},
+		Short:   "Testkube Cloud connect ",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			// create new cloud uris
+			opts.CloudUris = common.NewCloudUris(opts.CloudRootDomain)
+
+			client, _ := common.GetClient(cmd)
```

Comment:
shoud we check for error?

Reply:
it's not an error ;) it'll return namespace.